doc/specifications/cache/fos_http_cache.md

```diff
@@ -1,115 +1,31 @@
 # FOSHttpCacheBundle usage in eZ
 
-As of v5.4 / v2014.11, usage of [FOSHttpCacheBundle](http://foshttpcachebundle.readthedocs.org/) has been introduced,
-impacting the following features:
+Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
+features:
 
-* Http cache purge
+* Tags purging/baning (since version 1.8)
+* Http cache purge/ban
 * User context hash
 
 ## Http cache clear
 Varnish proxy client from FOSHttpCache lib is now used for clearing eZ Http cache, even when using Symfony HttpCache.
-A single `BAN` request is sent to registered purge servers, containing a `X-Location-Id` header.
-This header contains all Location IDs for which objects in cache need to be cleared.
+A single `BAN` request is sent to registered purge servers, containing a `xkey` header.
+This header contains cache items tags that need to be cleared.
```

Comment:
`item`, xkey _(side: and fastly)_ currently only supports clearing one tag per request, so several requests are made.